mne/viz/utils.py

```diff
@@ -43,6 +45,46 @@
                              'emg': "EMG sensor", 'ecog': "ECoG channel",
                              'misc': "miscellaneous sensor"}
 
+# Set the default backend
+default_backend = 'mlab'
+
+
+global _BACKEND
+try:
+    if _BACKEND is None:
+        _BACKEND = os.environ.get('MNE_3D_BACKEND', default_backend)
+except NameError:
+    _BACKEND = os.environ.get('MNE_3D_BACKEND', default_backend)
+
+
+def set_backend(backend_name):
```

Comment:
all these function names are a bit confusing to me (as a user). 

I guess that `set_backend`, `get_backend` etc are exposed in viz. Then the user code:

```py
from mne.viz import set_backend

set_backend(my_awesome_backend)
```
with this code I don't really understand which kind of backend. Either you add some info in the function name, or you put them in a directory.

```py
from mne.viz.3D import set_backend
set_backend(my_awesome_backend) # I know I'm talking aobut 3D 'cos I've been the one importing it.
```

```py
from mne.viz import set_3D_backend
set_3D_backend(my_awesome_backend) # I know I'm talking aobut 3D 'cos the name says so.
```

If you ask me I like the first one better.

Reply:
Good idea, thanks! I thought convenient to have it all in `mne.viz` to be honest so I can implement your second idea and wait for more feeback.

Reply:
Don't rush it.

How is the structure of `mne.viz`?